### PR TITLE
Add auto install dependences

### PR DESCRIPTION
**Pull request by laura-ding**
1. Add  third-party package for `fedora`, `centos7`, `centos6`, `ubuntu16`, `ubuntu18`
2. Add tool package for `fedora`

### CODE REVIEW

File: build_dep.sh [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+        xz-devel \
+        wget
+    wget $url_addr/fedora.tar.gz
+    tar xf fedora.tar.gz
+    pushd fedora
+        sudo rpm -ivh *.rpm
+    popd
+    wget $url_addr/vs-nebula-3rdparty.fc.x86_64.rpm
+    sudo rpm -ivh vs-nebula-3rdparty.fc.x86_64.rpm
+    echo "alias cmake='cmake -DNEBULA_GPERF_BIN_DIR=/opt/nebula/gperf/bin -DNEBULA_FLEX_ROOT=/opt/nebula/flex -DNEBULA_BOOST_ROOT=/opt/nebula/boost -DNEBULA_OPENSSL_ROOT=/opt/nebula/openssl -DNEBULA_KRB5_ROOT=/opt/nebula/krb5 -DNEBULA_LIBUNWIND_ROOT=/opt/nebula/libunwind ..'" >> ~/.bashrc
```
Comment (yixinglu) [inline build_dep.sh]: Is the `..` directory in the end of the alias command needed ?

Comment (yixinglu) [reply build_dep.sh]: Why not update `PATH` env variable to replace `alias cmake`?

Comment (laura-ding) [reply build_dep.sh]: has cmake args

**Comment by laura-ding** [comment]
jenkins go

**Comment by yixinglu** [comment]
closes #880

**Pull request merged**